src/technologies/f.json

```diff
@@ -446,16 +446,6 @@
     "scriptSrc": "cdn\\.usefathom\\.com/",
     "website": "https://usefathom.com"
   },
-  "Fbits": {
-    "cats": [
-      6
-    ],
-    "icon": "Fbits.png",
-    "js": {
-      "fbits": ""
```

Comment:
This isn't copied below. Looks like a few hundred sites are still emitting this signal. Is this js variable also used by the new Wake solution? If so, should it be added to the detection below? If not, would it be worth keeping this Fbits detection as well rather than delating it?